environments/india/postgresql.yml

```diff
@@ -4,15 +4,15 @@ REPORTING_DATABASES:
   aaa-data: aaa-data
 
 pgbouncer_override:
-  pgbouncer_default_pool: 290
+  pgbouncer_default_pool: 115
```

Comment:
This leaves room for the 5 reserved connections and 5 additional connections if we ever need direct connections to a database for debugging/firefighting.